sdk/monitor/azure-monitor-opentelemetry-exporter/samples/traces/sample_storage.py

```diff
@@ -0,0 +1,43 @@
+"""
+Examples to show usage of the azure-core-tracing-opentelemetry
+with the storage SDK and exporting to Azure monitor backend.
+This example traces calls for creating a container using storage SDK.
+The telemetry will be collected automatically and sent to Application
+Insights via the AzureMonitorTraceExporter
+"""
+
+import os
```

Comment:
How come this is a new file? Don't we already have a [sample_storage.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/sample_storage.py)?

Reply:
i think somehow i did not delete the old sample_storage previously - did that now